### ISSUE TITLE
Material can not deal with cpp pointer sign * in markdown environment which means italic

### ISSUE THREAD

**Issue by Daniel-UCAS** [atom-material/atom-material-syntax#47]
Example code
`
```cpp
const Foam::surfaceScalarField&
Foam::surfaceInterpolation::weights() const
{
    if (!weights_)
    {
        makeWeights();
    }

    return (*weights_);
}

```
`
The following * after weights_ will make contents after it italic
>  return (*weights_);
![2016-11-30_cpp_render](https://cloud.githubusercontent.com/assets/16081008/20738747/5163269a-b6f0-11e6-952c-c2aa891ad488.jpg)

**Comment by Daniel-UCAS** [comment]
This seems to be a common problem, just see
[language-gfm issues-44](https://github.com/atom/language-gfm/issues/44)
A simple solution is to put a space after *, but that is not perfect as expected
![2016-11-30_cpp_render_1](https://cloud.githubusercontent.com/assets/16081008/20752977/0e33ce16-b73f-11e6-925a-29fb50c312fb.jpg)

**Comment by silvestreh** [comment]
Yep, seems to happen with all syntax themes as I can reproduce this issue using One Dark/Light as well.

**Comment by Daniel-UCAS** [comment]
Is there a better way to solve this?

**Comment by silvestreh** [comment]
It seems that the space after the `*` character is the only way to go. Also, looks like the bug report on the language package is almost two years old which is really unfortunate it's not being prioritized.
